.rubocop_todo.yml

```diff
@@ -0,0 +1,423 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2019-09-25 10:06:45 -0700 using RuboCop version 0.74.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Cop supports --auto-correct.
```

Comment:
Are you cool with running auto-correct and then re-generating this file? That way we can knock out trivial exclusions.

Reply:
Done.